<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database</class>
 <widget class="QWidget" name="Database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="image_preview">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>303</width>
     <height>200</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="select_properties_layout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="property_row">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="gender_label">
        <property name="text">
         <string>Пол</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gender_combo">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>Мужской</string>
        </property>
        <item>
         <property name="text">
          <string>Мужской</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Женский</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="gender_label_2">
        <property name="text">
         <string>Контур глазной щели</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="countour_eye">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>Миндалевидный</string>
        </property>
        <item>
         <property name="text">
          <string>Миндалевидный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сегментоидный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Круглый</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="gender_label_3">
        <property name="text">
         <string>Степень раскрытия глазной щели</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="openin_eye_fissure">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="gender_label_4">
        <property name="text">
         <string>Положение глазной щели</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_4">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>Горизонтальное</string>
        </property>
        <item>
         <property name="text">
          <string>Горизонтальное</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Косовнутреннее</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Косонаружное</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="gender_label_5">
        <property name="text">
         <string>Положение век</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_5">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>Среднее</string>
        </property>
        <item>
         <property name="text">
          <string>Среднее</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="select_image_button">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Выбрать изображение</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>315</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttons_add_layout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Сохранить в Базу данных</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="header_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Глаза</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
